index.d.ts

```diff
@@ -48,6 +48,11 @@ declare namespace ansiStyles {
 		*/
 		readonly underline: CSPair;
 
+		/**
+		Make text overline. (Not widely supported)
```

Comment:
This should list the supporting terminals.

Reply:
e20eb46